### DISCUSSION TITLE
Some Images are not extracted from DOCX while using unstructured_client

### DISCUSSION THREAD

**Discussion by enx-github** [Unstructured-IO/unstructured#3414] [General]
Hi all,
I have used unstructured_client to extract text, images, and tables from my DOCX file. All the text and tables are extracted properly from the document, but some Images are not extracted from the document. 
I have written the code as below:
```
client = unstructured_client.UnstructuredClient(
    api_key_auth="xXx",
    server_url="https://api.unstructured.io/general/v0/general",
)
filename = "zZz"
file = open(filename, "rb")

response = client.general.partition(request=operations.PartitionRequest(
    partition_parameters=shared.PartitionParameters(
        # Note that this currently only supports a single file
        files=shared.Files(
            content=file.read(),
            file_name=filename,
        )
    ),
))
```

I am getting types of elements below:

- Header
- Image
- UncategorizedText
- NarrativeText
- Title
- PageBreak
- ListItem
- Table
- Footer

**Comment by enx-github** [comment]
The unstructured supports the image types below:
https://docs.unstructured.io/api-reference/api-services/overview

**Comment by Unicorn-1** [comment]
Does unstructured open source have image extraction from docx?
